### ISSUE TITLE
Fedora 25: ansible_selinux var

### ISSUE THREAD

**Issue by nusenu** [nusenu/ansible-relayor#44]
https://github.com/ansible/ansible-modules-core/issues/1797
https://github.com/ansible/ansible-modules-core/issues/1796

**Comment by nusenu** [comment]
ansible_selinux var problem:
https://github.com/ansible/ansible-modules-core/issues/2432

**Comment by nusenu** [comment]
For now we simply assume the default (selinux = enabled).

**Comment by nusenu** [comment]
the upstream bug got closed, lets test this again once ansible 0.2.4 is released

**Comment by nusenu** [comment]
on ansible 0.2.4.1.0 the variable is more accurate instead of False it says:

```
"ansible_selinux": {
            "status": "Missing selinux Python library"
        }
```
our current check for the SELinux dependencies and regathering facts work fine and will need to stay even after upstream fixed this now.

https://github.com/nusenu/ansible-relayor/blob/master/tasks/rpm_prepare.yml#L9
